mmv1/templates/terraform/examples/cloud_run_service_multiple_regions.tf

```diff
@@ -0,0 +1,230 @@
+# Cloud Run service replicated across multiple GCP regions
+
+######################################
+# Terraform (Project-wide resources) #
+######################################
+data "google_project" "app" {
```

Comment:
i don't think we need this here, i think we can use the id wherever it is referenced in the resources.

Reply:
Removed.